src/components/atoms/picker-sheet/index.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
-import { StyleSheet } from 'react-native';
+import { StyleSheet, Picker } from 'react-native';
 
 // React Native's own picker is deprecated, this is recommended and supported in expo SDK 38
 // Import from @react-native-community/picker/js/Picker to avoid warnings due to
 // https://github.com/react-native-community/react-native-picker/issues/109
-import Picker from '@react-native-community/picker/js/Picker';
+// remove because storybook does not work with it
+// https://github.com/react-native-community/react-native-picker/issues/45#issuecomment-633163973
+// import Picker from '@react-native-community/picker/js/Picker';
```

Comment:
@AlanSl 
>So we should avoid using it if we can. It also has some weird display issues on Android > that the non-deprecated one doesn't have.

>What's the problem with Storybook? Do you still get the error if you run expo update?
Storybook did not work with `@react-native-community/picker` because it could not find compiled version and importing es6 from node_modules did not work. I'll try again with `expo update` will see.

Reply:
Right, I had the same thing with Jest, it should be fixable with Webpack config.

It's happening because we're exporting the subcomponent instead of the main index to work around  https://github.com/react-native-community/react-native-picker/issues/109 - when that bug is fixed we can just import the main export which should give us the ES5 build.

In the meantime we can just tell Webpack to transpile this module. 

Something like one of these under `babel-loader` in the storybook webpack config:

```
    exclude: /node_modules/(?!(@react-native-community/picker/js/.+\\.jsx?))/
```

...and/or:

```
    include: /node_modules/@react-native-community/picker/js/.+\\.jsx?/
```
I think it won't be picked up automatically by the default Expo config because we're not using the default import path to avoid that bug.

Reply:
@AlanSl, yeah I did try to change webpack config to include `@react-native-community/picker` but I had no luck with that.

Reply:
RN's picker still there in react-native@0.62.2 `node_modules/react-native/Libraries/Components/Picker/Picker.js` and I don't see any reasons why not to use it for now.